### PR TITLE
[website] Add PWA feature to website

### PR DESCRIPTION
**Pull request by liyubin117**
### Purpose

Linked issue: close https://github.com/alibaba/fluss/issues/229
Support PWA (Progressive Web App) feature for website to make experience more user-friendly.

### Tests
image
image
image

### CODE REVIEW

**Comment by liyubin117** [comment]
@wuchong Hi, I found that Apache Flink, Paimon websites already support PWA feature, the feature is really practical for users, PTAL, thanks!

File: website/docusaurus.config.ts [hunk 1]
```diff
@@ -81,7 +81,24 @@ const config: Config = {
...
+          debug: true,
+          offlineModeActivationStrategies: [
+            'appInstalled',
+            'standalone',
+            'queryString',
+          ],
+          pwaHead: [
+            { tagName: 'link', rel: 'icon', href: '/img/logo.svg' },
+            { tagName: 'link', rel: 'manifest', href: '/manifest.json' },
+            { tagName: 'meta', name: 'theme-color', content: '#12affa' },
```
Comment (wuchong) [inline website/docusaurus.config.ts]: use the Fluss primary color `#0071e3`?

**Review by wuchong — COMMENTED** [review]
How to debug in local? I couldn't find the button on Chrome URL even after I installed the npm package.

**Comment by liyubin117** [comment]
> How to debug in local? I couldn't find the button on Chrome URL even after I installed the npm package.

use the following commands:
```
npm run build
npm run serve
```

File: website/docusaurus.config.ts (hunk 1 above)
Comment (liyubin117) [reply website/docusaurus.config.ts]: done:)

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**